test/inductor/test_torchinductor.py

```diff
@@ -3013,8 +3013,6 @@ def fn(w, i, o):
             (torch.randn([10, 4]), torch.randint(10, [8]), torch.tensor([0, 2, 6])),
         )
 
-    # FIXME: https://github.com/ROCmSoftwarePlatform/frameworks-internal/issues/3597
-    @skipIfRocm
     def test_batch_norm_2d(self):
```

Comment:
If we don't upload the miopen batch norm, this would stay skipped correct?

Reply:
When the fallback is created in `lowering.py` this UT will pass. Needs some investigation into why this is the case. (Details in above comment)